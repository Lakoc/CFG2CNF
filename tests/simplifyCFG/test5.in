S,A,B
a,b,c
S
S->B
B->A
B->c
A->ab
A->b
