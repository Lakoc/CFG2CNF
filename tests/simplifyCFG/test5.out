S,A,B
a,b,c
S
B->c
A->ab
A->b
B->b
B->ab
S->c
S->b
S->ab
